### ISSUE TITLE
Values do not fit in their text box

### ISSUE THREAD

**Issue by salibaj2** [FrSkyRC/ETHOS-Feedback-Community#3660]
X18SE, v1.5.2

As per this post: - https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/3554

text that does not fit in it's text box is all over the place. Another one in telemetry area.

![IMG_20240317_194009](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/68815554/9a41657d-e722-4b30-8da4-995acd86d25d)

**Comment by Richrfl** [comment]
I do not know what kind of aircraft are you flying, but I wonder why you want GPS speed in thousands of a Knot. Considering that the GPS data is received by the transmitter about 5 seconds late, make the numbers based on your aircraft speed and find out how much it will travel in 5 seconds, and review how many decimals you really need in your GPS speed. Another way to figure that out is to read the log (using the 3 decimals and disregarding if it will show the number or not) and find out the variation of the second and third decimals between readings.

**Comment by salibaj2** [comment]
I do not need any decimals for GPS, but it is not nice to have a text box with text that does not fit. Prior to version 1.4.1, I think, there was a similar situation.

Have you checked the VAR menu that I posted a link?

There is only 1 decimal, and if you look at the text box, from 100.0% you only see half a 0, 0.0%.

You think that is ok, because for me its not!

The first time I used VAR's i thought that there was something broken, as I could only see the unit changing.
